test/index.test.js

```diff
@@ -17,3 +17,20 @@ describe('Homepage', () => {
       });
   });
 });
+
```

Comment:
@sojida Good job but why is this test for office located within the index.test.js file? Why not have a file `office.test.js` and include all office related test there. It is cleaner to group tests into files with names relative to the test being done. This makes it easier for you to identify failing tests as your tests grow bigger

Reply:
That true. Thanks, I'll change it